### ISSUE TITLE
Make subject line more concise

### ISSUE THREAD

**Issue by lizzard** [mozilla/relman-auto-nag#629]
Minor nitpick. The subject lines are pretty long on these emails and I often can't see the relevant information (such as release version) on my laptop screen in list view without clicking through to the email. 

I suggest leaving out the words "which are" from the subject lines.  "Bugs which are tracked" can simply be "Bugs tracked"

**Comment by marco-c** [comment]
Agreed. Also, "[Using ML]" and "[autofix]", for the scripts for which they apply, can go to the end.

**Comment by calixteman** [comment]
Agree too
